src/accelerate/test_utils/testing.py

```diff
@@ -132,6 +133,16 @@ def require_fsdp(test_case):
     return unittest.skipUnless(is_torch_version(">=", "1.12.0"), "test requires torch version >= 1.12.0")(test_case)
 
 
+def require_torch_version(test_case=None, version=None):
```

Comment:
```suggestion
def require_torch_min_version(test_case=None, version=None):
```
Naming isn't completely right IMO.